### ISSUE TITLE
[Animations] `fadeOut` animation doesn't work for windows

### ISSUE THREAD

**Issue by 0x4nt** [hyprwm/Hyprland#849]
After trying to set up animations for windows, I've noticed that `fadeOut` animations doesn't work as exepected. 

## Current behavior
* Disable `windowsOut` animation to make it use `fade`
* Close a window
* Notice how it just disappear without any kind of animation.

## Expected behavior
The window disappears with a fade effect.

### Video
https://user-images.githubusercontent.com/93444783/195686443-72cb5b29-0153-429d-8dc0-d5c1182c9a74.mp4

### Configuration 
```
animations {
    enabled=1

    animation=windows,1,1.5,default
    animation=fadeOut,1,10,default
    animation=windowsOut,0,1,default
    animation=border,1,10,default
    animation=workspaces,1,3,default
}
```

### Hyprland version
```
Hyprland, built from branch main at commit 170def35d79210f9124cdfd7da563d5d384dcae9dirty (simplify shouldRenderWindow and fix one cond).
flags: (if any)
```

**Comment by vaxerski** [comment]
fixed in 670d6ce8f476cbdced7aa62e7d8c16a48bccf0d5

**Comment by vaxerski** [comment]
FYI: windowsOut works with fadeOut without any issues.
